xmake/modules/package/manager/clib/install_package.lua

```diff
@@ -0,0 +1,83 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015 - 2019, TBOOX Open Source Group.
+--
+-- @author      Adel Vilkov (aka RaZeR-RBI)
+-- @file        install_package.lua
+--
+
+-- imports
+import("core.base.option")
+import("core.project.config")
+import("lib.detect.find_tool")
+
+-- get configurations
+function configurations()
+    return
+    {
+        verbose = {description = "enable verbose output", default = "true", values = {"true", "false"}},
+        save = {description = "save dependency in project's package.json", default = "false", values = {"true", "false"}},
+        save_dev = {description = "save as development dependency in project's package.json", default = "false", values = {"true", "false"}},
+        out_dir = {description = "package installation directory relative to project root", default = "clib"},
```

Comment:
Please uses `default = false, type = "boolean"` instead of `default = "false", values = {"true", "false"}`